src/core/config/names.h

```diff
@@ -54,4 +54,13 @@ constexpr auto kIgnoreConstantCols = "ignore_constant_cols";
 constexpr auto kGraphData = "graph";
 constexpr auto kGfdData = "gfd";
 constexpr auto kMemLimitMB = "mem_limit";
+constexpr auto kOnlySfd = "only_sfd";
+constexpr auto kMinCard = "min_cardinality";
+constexpr auto kMaxDiffValsProportion = "max_different_values_proportion";
+constexpr auto kMinSfdStrengthMeasure = "min_sfd_strength";
+constexpr auto kMinSkewThreshold = "min_skew_threshold";
+constexpr auto kMinStructuralZeroesAmount = "min_st_zeroes_amount";
+constexpr auto kMaxFalsePositiveProbability = "max_false_positive_probability";
+constexpr auto kCordsDelta = "cords_delta";
```

Comment:
> Why we use `cords` prefix for variable name and for string literal?

It may be a good idea, since there are like 5 parameters for this algorithm.

Reply:
On the contrary, I am in favor of removing the prefix here too. We already have information about what options a particular algorithm uses, since we can call the corresponding method on the algorithm. Therefore, we usually omit the information that this is an option for a specific algorithm.

Reply:
> On the contrary, I am in favor of removing the prefix here too. We already have information about what options a particular algorithm uses, since we can call the corresponding method on the algorithm. Therefore, we usually omit the information that this is an option for a specific algorithm.

Ok, seems reasonable. Lets do like you suggest.